### PR TITLE
fix: remove console log

### PR DESCRIPTION
**Pull request by sylvesteraswin**
1. Remove `console.log` and use `debug`.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -68,9 +71,7 @@ function utilityPlugin(config?: PluginConfig) {
            */
           if (moduleName && !fs.existsSync(modulePath)) {
             // eslint-disable-next-line no-console
-            console.log(
-              chalk.red(`Module => ${moduleName} : is not supported`)
-            );
+            debug(`Module => ${moduleName} : is not supported`);
             process.exit(-1);
```
Comment (axyz) [inline src/index.ts:75]: should this also be 1 for consistency?

Comment (sylvesteraswin) [reply src/index.ts]: updated

**Comment by sylvesteraswin** [comment]
👍

**Comment by axyz** [comment]
:+1:

**Pull request merged**